README.md

```diff
@@ -8,7 +8,7 @@ The Ansible ``ansible.utils`` collection includes a variety of plugins that aid
 <!--start requires_ansible-->
 ## Ansible version compatibility
 
-This collection has been tested against following Ansible versions: **>=2.9.10,<2.11**.
+This collection has been tested against following Ansible versions: **>=2.9.10**.
```

Comment:
can we truly take off the upper limit? I understand we can test with 2.11, but not say 2.12, 2.13, etc?  I don't know for sure so check with someone more in the know :-)

Reply:
ack